ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/BearerTokenAuthHandler.java

```diff
@@ -24,32 +24,36 @@ public BearerTokenAuthHandler(TokenValidator validator) {
 	@Override
 	public void handle(Context ctx) throws Exception {
 		String authHeader = ctx.getRequest().getHeaders().get(HttpHeaderNames.AUTHORIZATION);
-		if (authHeader != null && authHeader.startsWith("Bearer")) {
-			//Confirm token is valid
+		String token = ctx.getRequest().getQueryParams().get("authToken");
```

Comment:
In the platform api service that is under development, the url for getting a clientconn websocket is http://hostname/ws?authToken=XXXXX (a GET request). I'm using ratpack-bearer-auth to authenticate on that url. If necessary, I could change it to a POST and use the conventional Bearer auth header, but I was hoping to not have to do that.